.github/workflows/check.yml

```diff
@@ -0,0 +1,29 @@
+name: Gradle Check
+on: [pull_request]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: adopt
+      - name: Run Gradle
+        run: |
+          echo "./gradlew check --no-daemon --no-scan"
+          ./gradlew check --no-daemon --no-scan
```

Comment:
Since we have multiple projects, we can leverage `--parallel`, it will make the runs faster.
Ref: https://docs.gradle.org/current/userguide/performance.html

Reply:
Makes sense. Let me test it on my local.